ControlExtensions/src/test/java/ControlExtensions/Angular/MultiSelect.java

```
@@ -0,0 +1,67 @@
+package ControlExtensions.Angular;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class MultiSelect extends ControlExtension implements ControlExtensions.MultiSelect {
+	Select menu;
+	List<String> selectedList = new ArrayList<>();
+
+	public MultiSelect(WebElement mappedElement) {
+		super(mappedElement);
+	}
+
+	@Override
+	public void selectAll(String[] selectees) {
+		this.menu = new Select(this.mappedElement);
+
+		if (this.menu.isMultiple() == true) {
+			try {
+				for (String optionName : selectees) {
+					this.menu.selectByVisibleText(optionName);
+				}
+			} catch (NoSuchElementException e) {
+				e.printStackTrace();
```

Don't swallow Exceptions. All Exceptions should stop code execution. 

This is essentially using Exceptions to test for conditions. Find an alternative technique.